core/trino-grammar/pom.xml

```diff
@@ -5,7 +5,7 @@
     <parent>
         <groupId>io.trino</groupId>
         <artifactId>trino-root</artifactId>
-        <version>423-SNAPSHOT</version>
+        <version>424-SNAPSHOT</version>
```

Comment:
let's have separate PR and merge it faster

Reply:
https://github.com/trinodb/trino/pull/18640